docs/issues/Issues.md

```diff
@@ -0,0 +1,37 @@
+#Issues
+
+##Labels
+All labels available: https://github.com/HeRAMS-WHO/herams-backend/labels
+
+##Creating an issue
+###Description
+When creating an issue be descriptive as possible. A good approach is:
+
+"I am logged in as `...` with `...`, and am on page `...`. I want to `...`"
+
+From this description it should be clear for developers to reproduce the taken steps to get to the problem.
+The desired next steps and features should be just as clear.
+
+###Assigning labels
+Labels must be assigned to make it very clear what the status of the issue is. Use the following issue categories:
+* **env** Indicate on which platform the error occurs, not needed when it is a new feature  
+* **urgency** When the urgency is other than normal, assign an urgency label like _extreme_, _high_ or _low_
```

Comment:
Default should be low. We could even automate this.

Reply:
I would say you want to use low to indicate an even lower as normal priority.